frontend/packages/dev-console/src/components/QuickStartTile.scss

```
@@ -1,6 +1,6 @@
 @import '../../../../public/style/vars';
 
-.odc-guidedtour-tile {
+.odc-quickstart-tile {
```

To be consistent with the rest of the changes:

```suggestion
.odc-quick-start-tile {
```